src/components/Records/FiltersChip.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <div class="chipsElements container-fluid">
+      <div class="row">
+        <div
+          v-for="(chip, index) in getChips"
+          :key="'Chips_' + index"
+        >
+          <div class="chip" v-for="(chipValue, subKey) in chip.paramVal" :key="'Chip_' + subKey">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Attribute ":key" should go before "class".](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=5022882)